floss/main.py

```diff
@@ -390,7 +399,8 @@ def load_vw(
     else:
         vw = viv_utils.getWorkspace(str(sample_path), analyze=False, should_save=False)
 
-    viv_utils.flirt.register_flirt_signature_analyzers(vw, list(map(str, sigpaths)))
+    if file_type == SUPPORTED_FILE_MAGIC["ELF"]:
```

Comment:
how about we make these explicit, so `SUPPORTED_FILE_MAGIC_ELF` and the same for `PE`?

Reply:
Yes I will add two constant with ELF and PE.